test/integration/targets/win_xml/tasks/main.yml

```diff
@@ -71,3 +71,49 @@
   assert:
     that:
     - not attribute_changed_result is changed
+
+- name: copy test files (https://github.com/ansible/ansible/issues/48471)
+  win_copy:
+    src: plane.zip
+    dest: "{{ win_output_dir }}\\plane.zip"
+
+- name: unarchive the test files
+  win_unzip:
+    src: "{{ win_output_dir }}\\plane.zip"
+    dest: "{{ win_output_dir }}\\"
+
+- name: change a text value UTF-8
+  win_xml:
+    path: "{{ win_output_dir }}\\plane-utf8.xml"
+    xpath: '/plane/year'
+    type: text
+    fragment: '1988'
```

Comment:
Just wondering if the fragment should include a non 7bit string (i.e. something like è as in the original bug report )?  I guess the characters that test this could be in the file already but it might make the purpose of the test clearer.

Reply:
@jhawkesworth made changes to names of the files and the names of the  tasks to  more  clearly represent the test's intent. Hope that is more clear.